### DISCUSSION TITLE
Add radius to FlBorderData ?

### DISCUSSION THREAD

**Discussion by dpaniagua0** [imaNNeo/fl_chart#1125] [Q&A]
Hello.

I was wondering if there is a way to customize the border.
For example adding a radius to the border. [ red border on the picture]

Thank you.
![Screenshot_20220820-192658](https://user-images.githubusercontent.com/1273602/185770590-fd789653-0b5f-4bd0-b5a0-a10d45f77c27.jpeg)

**Comment by imaNNeo** [comment]
There is no way to do that at this time.
btw, we can add this feature in the future.

Can you please create an issue for this feature request?
